src/test/components/TooltipCallnode.test.js

```diff
@@ -54,12 +54,12 @@ describe('TooltipCallNode', function () {
   }
 
   it('handles native allocations', () => {
-    const { profile, funcNamesDict } =
-      getProfileWithUnbalancedNativeAllocations();
+    const {
+      profile,
+      funcNamesDict: { A, B, Fjs, Gjs },
+    } = getProfileWithUnbalancedNativeAllocations();
     const threadIndex = 0;
-    const callNodePath = ['A', 'B', 'Fjs', 'Gjs'].map(
-      (name) => funcNamesDict[name]
-    );
+    const callNodePath = [A, B, Fjs, Gjs];
```

Comment:
It's nice to use the func names as variable directly instead of adding arbitrary strings!